intelligence_asn_test.go

```diff
@@ -0,0 +1,110 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+const TestASNNumber = "13335"
```

Comment:
we don't want this constant exported

```suggestion
const testASNNumber = "13335"
```